code/api/src/modules/user/mutations.js

```diff
@@ -49,6 +49,21 @@ export const userUpdate = {
       type: GraphQLInt
     },
 
+    name: {
+      name: 'name',
+      type: GraphQLString
+    },
+
+    role: {
+      name: 'role',
+      type: GraphQLString
+    },
+
+    twitter: {
+      name: 'twitter',
+      type: GraphQLString
+    },
```

Comment:
This seems strange to include a mutation for these. A user shouldn't be able to update their role, nor do we have twitter in our MVP